src/qml/pages/onboarding/Onboarding00Welcome.qml

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+// The Onboarding00Welcome page.
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+
+Column {
+    width: appTitle.width
+    spacing: 16
+
+    Image {
+        id: appLogo
```

Comment:
21c969272e718e63d3de00a9fe9e0f702eb4fd9f

Let's avoid unnecessary names (`id:`).

I also think these should be snake_case to distinguish from property names.

Reply:
> Let's avoid unnecessary names (`id:`).

Agree.

> I also think these should be snake_case to distinguish from property names.

I'd prefer to follow QML style provided by Qt docs. It guarantees more consistency in long run with more contributors.

Reply:
> Let's avoid unnecessary names (`id:`).

[Done](https://github.com/bitcoin-core/gui-qml/pull/43#issuecomment-933918510).